gendiff/scripts/gendiff.py

```diff
@@ -1,5 +1,34 @@
 #!/usr/bin/env python3
 import argparse
+import json
+
+
+def generate_diff(first_file: str, second_file: str):
+    f1 = json.load(open(first_file, "r", encoding="utf8"))
```

Comment:
it's better to do file operations in context managers, e.g. `open` can be used as one:

```python
with open(...) as file:
    ...
```